perception/point_containment_filter/include/moveit/point_containment_filter/shape_mask.h

```diff
@@ -133,6 +137,10 @@ class ShapeMask
   std::set<SeeShape, SortBodies> bodies_;
   std::map<ShapeHandle, std::set<SeeShape, SortBodies>::iterator> used_handles_;
   std::vector<bodies::BoundingSphere> bspheres_;
+
+  bool visualize_scaled_bodies_;
+  ros::NodeHandle* param_nh_;
```

Comment:
Don't use a pointer here - you can just instantiate this in the constructor - param_nh_("~").